homework-15/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework 15</title>
+</head>
+
+<body id="body">
+    <div class="container">
+        <label for="enter-city">
+            <select type="text" id="enter-city">

```

Comment:
Type is a redundant attribute
```suggestion
            <select id="enter-city">

```